plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/business-verification/index.tsx

```diff
@@ -35,7 +38,21 @@ export const BusinessVerificationStep: React.FC = () => {
 	};
 	const hasTestAccount = currentStep?.context?.has_test_account ?? false;
 
-	const handleStepChange = () => window.scroll( 0, 0 );
+	const subStepsList = [ 'activate', 'business', 'embedded' ];
+	// Only include the activate step if the user doesn't have a test account.
+	if ( ! hasTestAccount ) {
+		subStepsList.splice( 0, 1 );
+	}
```

Comment:
This is a nitpick, as directly mutating a variable can trigger a refresh in React, so it's better to avoid that.

I came up with this, but feel free to adjust if needed:

```suggestion
	const subStepsList = [ [
		...(!hasTestAccount ? [] : ['activate']), // Only include the activate step if the user doesn't have a test account.
		'business',
		'embedded'
];
```

Reply:
Good call. Made the changes.